ethcore/src/client/client.rs

```diff
@@ -1541,15 +1541,15 @@ impl EngineInfo for Client {
 }
 
 impl BlockChainClient for Client {
-	fn replay(&self, id: TransactionId, analytics: CallAnalytics) -> Result<Executed, CallError> {
+	fn replay(&self, id: TransactionId, analytics: CallAnalytics) -> Result<(Option<H256>, Executed), CallError> {
```

Comment:
Why is this Option? Isn't replay_block_transactions always return `Some(tx_hash)`?